### DISCUSSION TITLE
Display MultiILS instance name instead of code

### DISCUSSION THREAD

**Discussion by Londheamar** [vufind-org/vufind#4014] [General]
Hello Everyone,

I have installed VuFind and connected more than 5 Koha instances using the multi-backend driver.

However, when I attempt to log in, the driver names specified in the `multibackend.ini` file are displayed in the login dropdown. Instead of showing these codes, I would like to display the full library name.

Here’s a snippet from my `MultiBackend.ini` file:

```
[Drivers]
Koha1 = KohaILSDI
koha2 = KohaILSDI

; This section defines the driver instances that are enabled for login.
; Used together with MultiILS authentication method that allows selecting
; the login target.

[Login]
;default_driver = instance2
drivers[] = koha1
drivers[] = koha2
```

UI : Login form

![image](https://github.com/user-attachments/assets/0100acd2-cc0f-42e4-9568-8ee398c5a573)

In the VuFind login dropdown, it currently displays "koha1" and "koha2". I would prefer to show the full library names instead of these codes.

Could you suggest a better way to achieve this?

**Comment by EreMaijala** [comment]
It's not quite obvious, but you can add translations for the drivers to the language files (e.g. local/languages/en.ini) with the `source_` prefix. Example:
`source_koha1 = "Library 1"`
